README.md

```diff
@@ -105,6 +105,24 @@ Sent from Mission Control to instruct the rover to drive like a tank with a spec
 - `left` - left component in [-1.0, 1.0], where positive means drive forward on the left side and negative means drive backward on the left side
 - `right` - right component in [-1.0, 1.0], where positive means drive forward on the right side and negative means drive backward on the right side
 
+## Holonomic Drive Request
+### Description
+Sent from Mission Control to instruct the rover to have holonomic drive capabilities, such that it can rotate independently without requiring translational motion.
```

Comment:
I think it makes more sense to say something like: "Sent from Mission Control to instruct the rover to drive with holonomic capabilities (i.e., it can rotate independently without requiring translational motion)." The current wording makes it sound like a request to toggle a Boolean value.